### ISSUE TITLE
Message showing on Discover page

### ISSUE THREAD

**Issue by Thellie52** [ITPrism/CrowdFunding#252]
Hi,
I know that certain specific modules should only show on the project details page, but in the Crowdfunding instructions it says to select the Discover menu item... when I do this, it shows the following on the Discover page, "You should display this module only on the page with project details." If I don't select the Discover item, it doesn't show the module on the project details page...

Shouldn't it ignore the module for this page using the plugin "System - Crowdfunding Modules"?

...so how do I show the module on the correct page, and avoid the message on the Discover page? I'm running the latest version of Crowdfunding on a template called Lt_Charity.

Is it a bug in the new version, or something I've done incorrectly?

![discover](https://user-images.githubusercontent.com/39636072/40568252-b186bcae-60a3-11e8-9a2f-770c130dba55.png)
![project-page](https://user-images.githubusercontent.com/39636072/40568259-bc1a8060-60a3-11e8-8119-009b94fab982.png)

Thanks

**Comment by Thellie52** [comment]
I've searched the error message online, and found 4 other websites which have the same problem:
https://www.greatertomorrow.org/my-information?view=registration
http://www.geevit.com/index.php/en/discover
http://pasosdecumbia.tv/en/proyectos/45-prensa/19-el-libro-de-pasos-de-cumbia/embed
http://haitianhollywood.com/events.html

**Comment by Thellie52** [comment]
Okay, no need to bother answering... I've found multiple issues with this component, and no time to faff around or wait for an answer - having tried here and direct contact - ... the component is also no longer on the JED.

So, I've dumped Crowdfunding and am using a simple component called CMDonation instead.

Shame as it has huge potential for our foundation, and the project I'm involved in particular, but needs must.
